models/app.model.js

```diff
@@ -0,0 +1,59 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db
+    .query("SELECT slug, description FROM topics;")
+    .then((res) => {
+      return res.rows;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+exports.selectArticleId = (article_id) => {
+  return db
+    .query(
+      "SELECT author, title, article_id, body, topic, created_at, votes FROM articles WHERE article_id = $1;",
```

Comment:
You don't need to name all of the columns, you can just do SELECT * FROM articles WHERE article_id = $1;